### ISSUE TITLE
Search Current File command doesn't work on mobile

### ISSUE THREAD

**Issue by zcysxy** [ceciliamay/obsidianmd-theme-primary#31]
On mobile, the `Search Current File` window covers the whole screen, thus we cannot see the result. This is a bug that doesn't appear using `this.app.emulateMobile(true)` on desktop

**Comment by ceciliamay** [comment]
Hello there! To be correct is this the `Find` / `Find... Replace...` option in the options menu for a note? ((Hehe, clearly still an Obsidian noob))

**Comment by zcysxy** [comment]
Right! After the update it no longer covers the whole screen, and the size of the Find & Replace window is perfect on iPad. But it may be a little small on phones
![20220127_101937996_iOS](https://user-images.githubusercontent.com/48207708/151339782-3e4e6745-7240-4ee8-98b3-1cfdc15c746d.png)

**Comment by ceciliamay** [comment]
Noted! I'll try fixing this in the next update. Thanks for reporting!!
